projects.json

```diff
@@ -2346,14 +2274,6 @@
       {
         "version": "4.2",
         "commit": "2fe5b325a8a54e77c545b2f511213420da133b8c"
-      },
```

Comment:
Is it intended that we are only left with the 4.2 branch for Result instead of the 5.1 branch? I think we are now testing compiler `main` against the 4.2 branch of `Result` (which is causing a stress tester failure because offsets have moved around).

Reply:
Yeah, all of the packages where just defaulting to what either 4 or 5 before and once I added -swift-version support all of the ones I removed started failing with unavailable methods and such, if we want to add swift-version 5 back, need to figure out the commit for it…